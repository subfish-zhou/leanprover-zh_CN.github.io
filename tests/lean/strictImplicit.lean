def g {α : Type}   (a : α) := a
def f {{α : Type}} (a : α) := a

#check g
#check f
#check g 1
#check f 1
