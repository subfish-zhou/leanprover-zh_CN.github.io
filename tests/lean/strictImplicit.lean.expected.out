g : ?m → ?m
f : ⦃α : Type⦄ → α → α
g 1 : Nat
f 1 : Nat
